XRTK-Core/Packages/com.xrtk.core/Runtime/Services/MixedRealityToolkit.cs

```diff
@@ -316,10 +316,11 @@ void OnPlayModeStateChanged(UnityEditor.PlayModeStateChange playModeState)
                 }
 #endif // UNITY_EDITOR
 
-                EnsureMixedRealityRequirements();
-
+                // if the Toolit has a profile, validate toolkit requirements and initialise services
                 if (HasActiveProfile)
                 {
+                    EnsureMixedRealityRequirements();
```

Comment:
There's no point in putting this within the check.
If you've already gotten here then we've initialized the service locator and this needs to run

Reply:
Except if there is no profile, then is should not run.  Because it then causes other issues because it is running with no profile

Reply:
See https://github.com/XRTK/com.xrtk.core/pull/919#discussion_r800188587